owasp-top10-2017-apps/a3/insecure-go-project/README.md

```diff
@@ -30,7 +30,7 @@ WORKING!
 
 ## Attack Narrative
 
-To understand how this vulnerability can be exploited, check this section!
+To understand how this vulnerability can be exploited, check [this section]!
```

Comment:
Suggestion: What do you think of `[this section](docs/ATTACK.md)`?

Reply:
Thanks @rafaveira3 ! But the link to `[this section]` is at the end of the file!